Declare cupl_reset_write and cupl_eol_write function parameters to fix:
./cupl.h:134:22: warning: this function declaration is not a prototype [-Wstrict-prototypes]
./cupl.h:135:20: warning: this function declaration is not a prototype [-Wstrict-prototypes]

Remove first parameter of cupl_min and cupl_max to match changes in monitor.c.
--- cupl.h.orig	2015-11-21 17:51:45.000000000 -0600
+++ cupl.h	2023-01-12 21:34:33.000000000 -0600
@@ -131,8 +131,8 @@
 extern value allocate_value(int rank, int i, int j);
 extern void deallocate_value(value *);
 
-void cupl_reset_write();
-void cupl_eol_write();
+void cupl_reset_write(void);
+void cupl_eol_write(void);
 void cupl_scalar_write(char *name, scalar quant);
 void cupl_string_write(char *s);
 
@@ -151,8 +151,8 @@
 extern value cupl_ln(value);
 extern value cupl_log(value);
 extern value cupl_sqrt(value);
-extern value cupl_max(value, value);
-extern value cupl_min(value, value);
+extern value cupl_max(value);
+extern value cupl_min(value);
 extern value cupl_rand(value);
 
 extern value cupl_det(value);
